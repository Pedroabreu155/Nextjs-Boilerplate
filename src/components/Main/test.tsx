import { render, screen } from '@testing-library/react';

import { Main } from './index';

describe('<Main/>', () => {
  it('should render a heading', () => {
    const { container } = render(<Main />);

    expect(
      screen.getByRole('heading', {
        name: /react avan√ßado/i,
      }),
    ).toBeInTheDocument();
    expect(container.firstChild).toMatchSnapshot();
  });
});
