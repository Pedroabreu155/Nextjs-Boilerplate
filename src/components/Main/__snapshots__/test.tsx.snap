// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Main/> should render a heading 1`] = `
<main
  class="styles__Container-sc-19t6oi2-0 jCfqIT"
>
  <img
    alt="Imagem de um átomo (logo do React) e título React avançado ao lado da imagem."
    class="styles__Logo-sc-19t6oi2-1 iLQFsA"
    src="/images/logo.svg"
  />
  <h1
    class="styles__Title-sc-19t6oi2-2 htdEPi"
  >
    React Avançado
  </h1>
  <h2
    class="styles__Description-sc-19t6oi2-3 dygwdp"
  >
    Typescript, NextJS e Styled Components
  </h2>
  <img
    alt="Ilustração de um programador"
    class="styles__Illustration-sc-19t6oi2-4 bBVBdo"
    src="/images/hero-illustration.svg"
  />
</main>
`;
